src/tests/utilities.test.js

```diff
@@ -16,4 +20,11 @@ describe("basic utilities", () => {
 
     expect(result).toEqual(originalArrayBuffer);
   });
+
+  it("should convert an ArrayBuffer to Hex and back. Result should be the equal", async () => {
+    const original = new Uint8Array([0, 1, 10, 16, 1, 123, 33, 14, 15]);
```

Comment:
Im curious why you changed test data around, in my example we are testing the boundaries and it would have caught the incorrect radix used in the toHex function above